sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/DoubleExemplar.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.data;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.api.common.Attributes;
+import javax.annotation.concurrent.Immutable;
+
+/** An {@link Exemplar} with {@code double} measurments. */
+@Immutable
+@AutoValue
+public abstract class DoubleExemplar implements Exemplar {
+
+  public static DoubleExemplar create(
```

Comment:
we need some javadoc here, since it's a public method.

Reply:
Fixed